lib/zipkin-tracer/hostname_resolver.rb

```diff
@@ -19,6 +21,13 @@ def spans_with_ips(spans)
     LOCALHOST_I32 = 0x7f000001.freeze
     IP_FIELD = 3
 
+    def resolved_ip_address?(ip_string)
+      # When the ip_format is string, we will match with one of these two regexp
+      # When the ip_format is :i32 (used by kafka), we just check the string is a number
+      !!((ip_string =~ Resolv::IPv4::Regex) || (ip_string =~ Resolv::IPv6::Regex)) ||
```

Comment:
This is in preparation when we support ipv6. We will need a different pr for that